INSTALL.md

```diff
@@ -1,95 +1,508 @@
-# Prerequisites
+# Instructions
 
-| Linux | macOS | Windows 10
-| - | - | -
-| none | [Xcode Command Line Tools Package][xcode] | [Windows Terminal][terminal] and [Windows Subsystem for Linux (WSL)][wsl]
+These instructions explain how to set up the tools required to build **pokeemerald**, which assembles the source files into a ROM.
 
-[xcode]: https://developer.apple.com/library/archive/technotes/tn2339/_index.html
-[terminal]: https://docs.microsoft.com/windows/terminal/get-started
-[wsl]: https://docs.microsoft.com/windows/wsl/install-win10
+If you run into trouble, ask for help on Discord or IRC (see [README.md](README.md)).
 
-Independently from the specific OS, make sure that the `gcc`, `g++`, `make`, `git`, and `libpng-dev` packages or their equivalents are installed and accessible to the development tools that are used by the project (this means that, for example, on Windows, the packages have to be installed in the WSL environment). The package names and installation methods may vary with each OS.
+## Windows
+Windows has instructions for building with three possible terminals, if users encounter unexpected errors in following instructions for one of the terminals. These instructions are:
```

Comment:
Not sure I understand the wording here, of `if users encounter unexpected errors in following instructions for one of the terminals.` Doesn't sound like a complete thought. What exactly do you mean?

Reply:
There are multiple instructions in case a user tries following one set of instructions, but is unable to complete the installation. They're there as backups (and also in the case of msys2, there for Windows 7 and 8).

Reply:
Ok. I'm not sure we need the line after the comma (surely this isn't the only reason we include instructions for other terminals) but if you want to keep it it should be `terminals in case` instead of `terminals, if`. The latter was confusing to read.

Reply:
"Windows has instructions for building with three possible terminals, providing 3 different options in case the user stumbled upon unexpected errors." would be an alternative way to word the thought, and it's easier to understand as well.

Reply:
Went with Lunos's suggestion